massa-node/base_config/config.toml

```diff
@@ -1,96 +1,175 @@
 
 
 [logging]
+    # increase for more info about the node. May reduce your node ability to keep up with the network
     level = 2
 
 [api]
+    # max number of future periods considered during requests
     draw_lookahead_period_count = 10
+    # where to send admin level requests (node management)
```

Comment:
address:port on which the node API listens for admin and node management requests. Dangerous if publicly exposed.